### ISSUE TITLE
Have Cmd and Alt on POP!_OS 22.04 LTE

### ISSUE THREAD

**Issue by coloeus** [rbreaves/kinto#747]
Hi,
how can I change to the order of `Ctrl | Option | Command` on my POP!_OS installation?
When I change the keyboard layout in the OS to Apple Keyboard the other buttons like `{},[],...` are at the right position.

**Comment by rbreaves** [comment]
keyboard layout in your OS? You shouldn't be changing anything to an Apple keyboard from within the OS... Kinto simply takes care of everything, but no it doesn't let you configure the specific modifier key layout either.. it is easily changed though near the top of the file & a restart of kinto. `~/.config/kinto/kinto.py`.

And no idea about your symbols issue. What language is your keyboard?

**Comment by coloeus** [comment]
First of all thanks for you answer.
What I want to achieve is that for example I can copy with `Cmd+C` and Cmd is the first key left of spacebar.
Another thing that I'm interested in is that [ ] `brackets` and ( ) , { } are on the same position like they would be on macOs.
I'm a developer

**Comment by rbreaves** [comment]
Yes, I understand that and the app does do that afaik by default. So again - what is your default language, is it US English or something else? I dev on US English only, atm, and again if you had set anything keyboard related outside of the Kinto app itself then that could be causing you issues - especially if it relocated any of your modifier keys before running Kinto itself.

**Comment by coloeus** [comment]
Sorry, I read over your question. My default keyboard layout is: German (no dead keys).
Pressing Super/Windows-Key + C is Copy but the other keys aren't working like intended.
And how can I switch Super/Windows-Key with the button left of space like it's on mac?

**Comment by rbreaves** [comment]
> And how can I switch Super/Windows-Key with the button left of space like it's on mac?

That is what confuses me.. the setup typically has you press the 2nd key left of the spacebar which tells my app how to do correct remapping without the user needing to set it.. but even if the setup gets it wrong the Kinto GUI app or system tray lets you change your remaps and you can easily switch btwn the different modes (there are 4 major modes, but technically 5).

Generally though during the setup if you press the Windows key (Windows mode) during setup then it knows to remap physical Alt key to being Ctrl a majority of the time to make it feel like you are using a Cmd key. If you press Alt (Apple mode) as the 2nd key left of the Spacebar then I know you are already on an Apple keyboard & so then I remap Cmd to Ctrl to fake out the Cmd key most of the time.

The last keyboard type/mode is the IBM and chromebook keyboards without any sort of Win/Cmd key on the bottom row and I remap it as it should be as well.. Point being your question sorta makes no sense to me because unless you have modifier keys on that bottom row that doesn't fit any of these 4 major keyboard layouts then I am at a loss.

What key do you physically see as your 2nd key left of the Spacebar? Would providing a picture of it be useful? This is why I am suspecting you have done some sort of remapping of your modifier keys before Kinto ever gets to that modifier. And I don't understand why you have Super/Win providing a Copy function at all - not when you are using Kinto. Does regular Ctrl-C no longer work when you have remapped Win/Super-C to copy?
